app/main.py

```diff
@@ -1,13 +1,216 @@
 def format_linter_error(error: dict) -> dict:
```

Comment:
u need to make a function that will be able to work with different text like:
```
def format_linter_error(error: dict) -> dict:
    return {
    "line": error.get("line_number") 
    "column":  error.get("column_number"), 
    "message": ..., 
    "name": ..., 
    "source": ...
}
```
etc.